magma/lib/magma/identifier.rb

```diff
@@ -0,0 +1,7 @@
+class Magma
+  class Identifier < Sequel::Model
+     many_to_one :rule
+     many_to_one :renamed_to, class: self, key: :renamed_to_id
```

Comment:
Hm....is `:renamed_to` many to one, or one-to-one? Like ... you might have many identifiers renamed to the same one? That seems ...  hm, kind of mind boggling. Like, PAT1 -> PAT3, PAT2 -> PAT3, etc.?